indexer/locals.tf

```diff
@@ -166,6 +166,10 @@ locals {
             name : "S3_BUCKET_ARN",
             value : aws_s3_bucket.athena_rds_snapshots.arn,
           },
+          {
```

Comment:
If we want this on by default everywhere, we should just enable this loop in `roundtable/config.ts`, I think we should turn it off in dev environments though